test/Transforms/PagerandoWrappers/varargs.ll

```diff
@@ -1,18 +1,39 @@
 ; RUN: opt < %s -pagerando-wrappers -S | FileCheck %s
 
+; CHECK-LABEL: %struct.va_list = type { i8*, i8*, i8*, i32, i32 }
```

Comment:
No need to add this. We want the test to be as generic and minimal as possible while still checking for the functionality we're testing.